REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.5
+julia 0.7
```

Comment:
Is it fine to drop both Julia 0.5 and 0.6 support on master? Otherwise some compatibility additions are needed.